{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"mach_exception_subcode_t"},{"kind":"text","text":" = "},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/mach_exception_data_type_t","preciseIdentifier":"c:@T@mach_exception_data_type_t","text":"mach_exception_data_type_t"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/nio\/mach_exception_subcode_t"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.swift.docc\/documentation\/NIO\/mach_exception_subcode_t","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"mach_exception_subcode_t"}],"title":"mach_exception_subcode_t","roleHeading":"Type Alias","role":"symbol","symbolKind":"typealias","externalID":"c:@T@mach_exception_subcode_t","modules":[{"name":"NIO"}],"navigatorTitle":[{"kind":"identifier","text":"mach_exception_subcode_t"}]},"hierarchy":{"paths":[["doc:\/\/org.swift.docc\/documentation\/NIO"]]},"references":{"doc://org.swift.docc/documentation/NIO/mach_exception_subcode_t":{"role":"symbol","title":"mach_exception_subcode_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"mach_exception_subcode_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/mach_exception_subcode_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"mach_exception_subcode_t"}],"url":"\/documentation\/nio\/mach_exception_subcode_t"},"doc://org.swift.docc/documentation/NIO":{"role":"collection","title":"NIO","abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO","kind":"symbol","type":"topic","url":"\/documentation\/nio"},"doc://org.swift.docc/documentation/NIO/mach_exception_data_type_t":{"role":"symbol","title":"mach_exception_data_type_t","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"mach_exception_data_type_t"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/mach_exception_data_type_t","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"mach_exception_data_type_t"}],"url":"\/documentation\/nio\/mach_exception_data_type_t"}}}